fastmri/data/mri_data.py

```diff
@@ -301,6 +314,27 @@ def __init__(
                 if ex[2]["encoding_size"][1] in num_cols  # type: ignore
             ]
 
+    def Download_csv(self, version):
+        url = 'https://cdn.jsdelivr.net/gh/microsoft/fastmri-plus@{0}/Annotations/knee.csv'.format(version)
```

Comment:
Could you give me a reference on this? Is this the GitHub api?

Also, I recommend style recommendations to use f-strings instead of `.format` unless you have a strong reason for doing it this way.

Lastly, you're only considering the knee annotations. We should be able to consider the brain annotations too.

Reply:
Here's reference for: [jsdelivr](https://github.com/jsdelivr/jsdelivr). Home page: https://www.jsdelivr.com
jsdelivr is a CDN or reverse proxy.

For annotation type, should we give user options or decide it by file that user uploaded?

Reply:
Could we avoid using a third party platform for this? We could use `subprocess` to clone the repo to a temporary directory, copy the file, and then delete the directory. A brief description: https://stackoverflow.com/questions/51239168/how-to-download-single-file-from-a-git-repository-using-python

You could copy the file to some cache directory, such as `cwd / annotation_cache` and name it with the git hash, e.g., `cwd/annotation_cache/67ed9a68ee33d3197f93893e8125688470c4a577knee_annotation.csv`.

Yes the user should decide the annotations.

Reply:
Yes, we can clone the repo directly and do following git operation. But we have to add "git" as dependency. If it's acceptable, I can do it.